smrt/rtsolver/dort.py

```diff
@@ -638,8 +638,9 @@ def dort_modem_banded(
         if self.snowpack.substrate is None and optical_depth < 5:
             smrt_warn(
                 "DORT has detected that the snowpack is optically shallow (tau=%g) and no substrate has been set, meaning that the space "
-                "under the snowpack is vaccum and that the snowpack is shallow enough to affect the signal measured at the surface."
-                "This is usually not wanted. Either increase the thickness of the snowpack or set a substrate."
+                "under the snowpack is empty, the last layer bottom interface might be missing and"
```

Comment:
please remove "the last layer bottom interface might be missing", this is ambiguous, and not necessary given the remaining of the message.